runtime/list.c

```diff
@@ -39,6 +39,13 @@ static bool getFiller(ListDescPtr desc, TaggedPtr *valuePtr) {
     return false;
 }
 
+// must be called with index less than lp->gArray.length
```

Comment:
And >= 0 presumably.

I would like to follow the convention that comments on a line by themselves start with a capital letter, especially before a function.